### DISCUSSION TITLE
新增鼠标移动轨迹以及点击特效

### DISCUSSION THREAD

**Discussion by fightingHawk2001** [shuax/MouseInc.Settings#413] [Ideas]
开发者你好：
        能不能新增两个功能：
        1. 鼠标移动时增加鼠标移动轨迹（线条拖尾，像手势轨迹那样的就好）
        2. 鼠标点击时增加鼠标点击特效（光圈波纹扩撒，像一个光圈像水波纹一样网外扩撒）
        
        细节方面就要靠大佬您掌握了（颜色、宽度、距离等等），当然也不需要太花哨，能够看出来就行

**Comment by SoyKoa** [comment]
感觉有花哨了
